src/main/java/com/redhat/cloud/policies/app/model/history/PoliciesHistoryEntry.java

```diff
@@ -0,0 +1,101 @@
+package com.redhat.cloud.policies.app.model.history;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "policies_history")
+public class PoliciesHistoryEntry {
+
+    // The ID is generated by policies-engine.
+    @Id
+    private UUID id;
+
+    @Column(name = "tenant_id")
+    private String tenantId;
```

Comment:
Should we stop calling it `tenant_id` and start moving to `account_id` instead?

Reply:
This project contains mostly `tenantId` variables and only one `accountId` so I would stick with `tenantId` here.

Reply:
Lets stick to `tenantId` for now then.